pkgs/build-support/appimage/default.nix

```diff
@@ -37,14 +37,24 @@ rec {
   extractType2 = extract;
```

Comment:
Is it worth trying to do some analysis to see if there are exceptions? Or will we apply those in follow ups (if they exist)?

Reply:
I'm really hoping this is all just a first pass and maintainers will continually refine things. But I've left in the possibility of override so they can.